lib/jekyll/tags/include.rb

```diff
@@ -2,14 +2,6 @@
 
 module Jekyll
   module Tags
-    class IncludeTagError < StandardError
-      attr_accessor :path
-
-      def initialize(msg, path)
-        super(msg)
-        @path = path
-      end
-    end
 
```

Comment:
😞  You're right. I think this case is unlikely but nonetheless we shouldn't remove this then. I'll update the PR later, thank you!